spec/models/cost_query/filter_spec.rb

```diff
@@ -345,44 +352,40 @@ def update_work_package_custom_field(name, options)
         clear_cache
       end
 
-      def class_name_for(name)
-        "CostQuery::Filter::CustomField#{WorkPackageCustomField.find_by_name(name).id}"
-      end
+      include OpenProject::Reporting::SpecHelper::CustomFieldFilterHelper
```

Comment:
I don't know what ruby best practices have to say here, but I am not a huge fan of including in the middle of a class.